src/components/__tests__/Calculator.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calc from '../Calculator';
+
+describe('Home component', () => {
+  it('renders correctly', () => {
+    const tree = renderer
+      .create(<Calc />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Kindly write unit tests for all functionalities and scenarios in the operate.js file and the calculate.js file. 👍 When you write unit tests, bugs can be found easily and quicker. Unit tests help developers detect problems immediately, and then fix them quickly. With fewer resources spent finding bugs, teams can move on to the next phase of a project. Kindly implement this change. 💻 🖱️